web-app/src/main/java/com/infoshare/webapp/controller/GameController.java

```diff
@@ -0,0 +1,80 @@
+package com.infoshare.webapp.controller;
+
+import com.infoshare.webapp.model.Answers;
+import com.infoshare.webapp.model.Game;
+import com.infoshare.webapp.model.Questions;
+import com.infoshare.webapp.service.GameService;
+import com.infoshare.webapp.service.QuestionService;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.util.List;
+
+@Controller
+public class GameController {
```

Comment:
Myślę, że zamiast używania Long możesz zmienić na long. Zwiększy to wydajność aplikacji.